prac_05/emails.py

```diff
@@ -0,0 +1,42 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual: 35 minutes
+"""
+
+def main():
+    email_to_name = {}
+    email = validate_email()
+    user_name = name_processing(email)
+    while email != "":
+        choice = input(f"Is your name {user_name}? (Y/n) ").upper()
+        if choice == "Y":
+            email_to_name[email] = user_name
+        else:
+            modified_name = input("Name: ")
+            email_to_name[email] = modified_name
+        email = validate_email()
+        user_name = name_processing(email)
+
+    for email in email_to_name:
+        print(f"{email_to_name[email]} ({email})")
+
+def name_processing(email):
```

Comment:
Function name should be written with verb_something, for eg. it should be "process_name_from_email" for this function.